account_tax_unece/data/unece_tax_categ.xml

```diff
@@ -20,57 +20,62 @@ Source : https://www.unece.org/fileadmin/DAM/trade/untdid/d16b/tred/tred5305.htm
+        <field name="description">
+            A tax category code indicating the item is tax exempt when the item is
+            bought for future resale.
+        </field>
     </record>
     <record id="tax_categ_ac" model="unece.code.list">
         <field name="type">tax_categ</field>
         <field name="code">AC</field>
         <field name="name">Value Added Tax (VAT) not now due for payment</field>
-        <field
-            name="description"
-        >A code to indicate that the Value Added Tax (VAT) amount which is due on the current invoice is to be paid on receipt of a separate VAT payment request.</field>
+        <field name="description">
+            A code to indicate that the Value Added Tax (VAT) amount which is due on the
+            current invoice is to be paid on receipt of a separate VAT payment request.
+        </field>
     </record>
     <record id="tax_categ_ad" model="unece.code.list">
         <field name="type">tax_categ</field>
         <field name="code">AD</field>
         <field name="name">Value Added Tax (VAT) due from a previous invoice</field>
-        <field
-            name="description"
-        >A code to indicate that the Value Added Tax (VAT) amount of a previous invoice is to be paid.</field>
+        <field name="description">
+            A code to indicate that the Value Added Tax (VAT) amount of a previous
+            invoice is to be paid.
+        </field>
     </record>
     <record id="tax_categ_ae" model="unece.code.list">
         <field name="type">tax_categ</field>
         <field name="code">AE</field>
         <field name="name">VAT Reverse Charge</field>
-        <field
-            name="description"
-        >Code specifying that the standard VAT rate is levied from the invoicee.</field>
+        <field name="description">
+            Code specifying that the standard VAT rate is levied from the invoicee.
+        </field>
     </record>
     <record id="tax_categ_b" model="unece.code.list">
         <field name="type">tax_categ</field>
         <field name="code">B</field>
         <field name="name">Transferred (VAT)</field>
-        <field
-            name="description"
-        >VAT not to be paid to the issuer of the invoice but directly to relevant tax authority.</field>
+        <field name="description">
+            VAT not to be paid to the issuer of the invoice but directly to relevant tax
+            authority.
+        </field>
     </record>
     <record id="tax_categ_c" model="unece.code.list">
         <field name="type">tax_categ</field>
         <field name="code">C</field>
         <field name="name">Duty paid by supplier</field>
-        <field
-            name="description"
-        >Duty associated with shipment of goods is paid by the supplier; customer receives goods with duty paid.</field>
+        <field name="description">
+            Duty associated with shipment of goods is paid by the supplier; customer
```

Comment:
Are all of these changes done by pre-commit? If so, they must be in the pre-commit commit, not here.

Reply:
What is a pre-commit commit ? Does it mean that you would have prefered to see in a separate commit ?

Reply:
Please check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch where migration steps including pre-commit are included

Reply:
It is possible to skip the automatic reformatting of a particular file by adding its path in the `.prettierignore`, like we did here: https://github.com/OCA/l10n-netherlands/blob/13.0/.prettierignore

Reply:
The reformatting is good and should be done here, but on its separate commit according the procedure.

Reply:
The reformatting is not good.

Here for example we have:

```xml
        <field name="description">
            Duty associated with shipment of goods is paid by the supplier
```

which however does not behave the same as:


```xml
        <field
           name="description"
          >Duty associated with shipment of goods is paid by the supplier
```

If you want to realize it, open the unece.code.list tree view: you will notice that the text in the description column is misaligned in most of the records.

Since the automatic reformatting is forcing such unwanted changes, just include the path of the files in the .prettierignore so that the reformatting will be not executed.

Reply:
OK, then it's good to know and indeed it affects in this case.

@Yajo is this something related with ignore-whitespace things?

Reply:
Fix in https://github.com/OCA/oca-addons-repo-template/pull/35.